app/main.js

```diff
@@ -18,17 +18,40 @@ commander.program
   .option('-a, --all', 'Scrape all airports in all the countries')
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
+const listDisabled = (playbooksDir) => {
```

Comment:
I know dividing stuff into functions is good but rest of the main.js is all one big block with no functions. Can you also make it similar to the existing code.